pkg/sdn/plugin/proxy.go

```diff
@@ -98,7 +103,39 @@ func (proxy *OsdnProxy) watchEgressNetworkPolicies() {
 	})
 }
 
+func (proxy *OsdnProxy) watchNetNamespaces() {
```

Comment:
blah... we need to abstract out the EgressNetworkPolicy code better so that node and proxy can share more code rather than duplicating. Can you add a TODO comment for that?